### PR TITLE
Move uses to mainnet_blocks_from_rpc

### LINKED ISSUE #427: Move dedicated `use`s into `mainnet_blocks_from_rpc`
To remove these warnings when running `cargo test` (without the `rpc-storage` feature) in `main`:
```
warning: unused imports: `ProviderBuilder` and `Provider`
 --> crates/pevm/tests/mainnet.rs:3:22
  |
3 | use alloy_provider::{Provider, ProviderBuilder};
  |                      ^^^^^^^^  ^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused imports: `BlockId` and `BlockTransactionsKind`
 --> crates/pevm/tests/mainnet.rs:4:23
  |
4 | use alloy_rpc_types::{BlockId, BlockTransactionsKind};
  |                       ^^^^^^^  ^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `reqwest::Url`
 --> crates/pevm/tests/mainnet.rs:5:5
  |
5 | use reqwest::Url;
  |     ^^^^^^^^^^^^

warning: unused import: `PevmChain`
 --> crates/pevm/tests/mainnet.rs:7:19
  |
7 | use pevm::chain::{PevmChain, PevmEthereum};
  |                   ^^^^^^^^^
```

### PR DESCRIPTION
**Pull request by fiplox**
move some uses to remove annoying warnings

Closes #427

### CODE REVIEW

File: crates/pevm/tests/mainnet.rs [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 #[cfg(feature = "rpc-storage")]
 async fn mainnet_blocks_from_rpc() {
+    use alloy_provider::{Provider, ProviderBuilder};
+    use pevm::chain::PevmChain;
+
     let rpc_url = match std::env::var("ETHEREUM_RPC_URL") {
         // The empty check is for GitHub Actions where the variable is set with an empty string when unset!?
         Ok(value) if !value.is_empty() => value.parse().unwrap(),
-        _ => Url::parse("https://eth.public-rpc.com").unwrap(),
+        _ => reqwest::Url::parse("https://eth.public-rpc.com").unwrap(),
```
Comment (hai-rise) [inline crates/pevm/tests/mainnet.rs:16]: We should indeed quantify for general names like `Url`!

File: crates/pevm/tests/mainnet.rs [hunk 2]
```diff
@@ -35,14 +34,20 @@ async fn mainnet_blocks_from_rpc() {
     ] {
         let provider = ProviderBuilder::new().on_http(rpc_url.clone());
         let block = provider
-            .get_block(BlockId::number(block_number), BlockTransactionsKind::Full)
+            .get_block(
+                alloy_rpc_types::BlockId::number(block_number),
```
Comment (hai-rise) [inline crates/pevm/tests/mainnet.rs:38]: This is even longer than `alloy_provider`. For consistency can we `use` it as well 🙏?

**Review by hai-rise — CHANGES_REQUESTED** [review]
One small nit and thank you for contributing!

**Comment by fiplox** [comment]
Done :)
There are people that like/dislike full notation to be more verbose ;)

File: crates/pevm/tests/mainnet.rs [hunk 3]
```diff
@@ -41,8 +42,11 @@ async fn mainnet_blocks_from_rpc() {
             .unwrap();
         let chain = PevmEthereum::mainnet();
         let spec_id = chain.get_block_spec(&block.header).unwrap();
-        let rpc_storage =
-            pevm::RpcStorage::new(provider, spec_id, BlockId::number(block_number - 1));
+        let rpc_storage = pevm::RpcStorage::new(
+            provider,
+            spec_id,
+            alloy_rpc_types::BlockId::number(block_number - 1),
```
Comment (hai-rise) [inline crates/pevm/tests/mainnet.rs:48]: Can revert this now 🙏.

**Review by hai-rise — CHANGES_REQUESTED** [review]
Almost there!

File: crates/pevm/tests/mainnet.rs [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
 
 pub mod common;
 
 #[tokio::test(flavor = "multi_thread")]
 #[cfg(feature = "rpc-storage")]
 async fn mainnet_blocks_from_rpc() {
+    use alloy_provider::{Provider, ProviderBuilder};
+    use alloy_rpc_types::{BlockId, BlockTransactionsKind};
+    use pevm::chain::PevmChain;
+    use reqwest::Url;
```
[the remark this answers is not in the document]
Comment (fiplox) [reply crates/pevm/tests/mainnet.rs:13]: oups, read too fast.

File: crates/pevm/tests/mainnet.rs (hunk 3 above)
Comment (fiplox) [reply crates/pevm/tests/mainnet.rs:48]: This one wanted to stay so badly xD

**Review by hai-rise — APPROVED** [review]
Awesome, thank you!

**Pull request merged**